Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TwoHandManipulatableMode.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace HoloToolkit.Unity.InputModule.Utilities.Interactions
+{
+    /// <summary>
+    /// Sorting type for collections
+    /// </summary>
+    public enum TwoHandManipulatableMode
```

Comment:
We should probably update this to use the `[Flags]` attribute so we can do proper bit ops on this enum. It'll make working with that switch statement cleaner too.